admin/cleanup.py

```diff
@@ -65,6 +57,22 @@ def get_ec2_driver(aws):
     return ec2
 
 
+def _get_node_creation_time(node):
+    """
+    Get the creation time of a libcloud node.
+
+    Rackspace and EC2 store the information in different metadeta.
```

Comment:
:question: Perhaps comment on which one is EC2 and which is Rackspace ("launch_time" vs "created")